dataactbroker/scripts/dedup_duns_export.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import boto
+import os
+import pandas as pd
+import csv
+from datetime import datetime
+
+from dataactvalidator.health_check import create_app
+from dataactcore.logging import configure_logging
+from dataactcore.config import CONFIG_BROKER
+
+logger = logging.getLogger(__name__)
+
+# CSV column header name in DUNS file
+column_headers = [
+    "DUNS",
+    "Registration_Date",
+    "Expiration_Date",
+    "Last_Update_Date",
+    "Activation_Date",
+    "Legal_Business_Name"
+]
+
+
+def generate_dedup_export(duns_file, dedup_export_path):
+    """Generate dedup duns export"""
+
+    duns_df = pd.read_csv(duns_file, skipinitialspace=True, header=None, encoding='latin1', quotechar='"',
+                          dtype=str, names=column_headers, skiprows=1)
+    duns_modified_df = duns_df.drop_duplicates(subset=['DUNS'], keep='last')
+
+    return duns_modified_df.to_csv(dedup_export_path, columns=column_headers, index=False, quoting=csv.QUOTE_ALL)
```

Comment:
Why is this returning when it doesn't do anything with the return the only time it's called?

Reply:
Force of habit. Fixing.